doc/changes/latest.rst

```diff
@@ -30,4 +30,5 @@ Changes
 -------
 
 - :bdg-success:`API` Expose scipy CubicSpline ``extrapolate`` parameter in :func:`~signal.clean` to control the interpolation of censored volumes in both ends of the BOLD signal data (:gh:`4028` by `Jordi Huguet`_).
+- :bdg-success:`API` Switch to using tox to manage environments during development and testing. All plotting python dependencies (matplotlib AND plotly) are now installed when running ``pip install nilearn[plotting]`` (:gh:`4029` by `Rémi Gau`_).
```

Comment:
filing this under API... but not sure this is the best

Reply:
why not :bdg-secondary:\`Maint\` ?

Reply:
oh dear...
because I had forgotten we had it: fixed now

Reply:
Did you push the fix?

Reply:
I thought I had...

Reply:
I guess this way we can test the whole tox thing on a PR...
https://github.com/nilearn/nilearn/pull/4114